src/d/actor/d_a_tag_etc.cpp

```diff
@@ -4,50 +4,175 @@
 //
 
 #include "d/actor/d_a_tag_etc.h"
+#include "d/d_com_inf_game.h"
+#include "d/d_event.h"
+#include "d/d_event_manager.h"
 #include "dolphin/types.h"
+#include "f_op/f_op_actor_iter.h"
+#include "f_pc/f_pc_searcher.h"
+#include "global.h"
 
 /* 00000078-00000084       .text getEventNo__11daTag_Etc_cFv */
-void daTag_Etc_c::getEventNo() {
-    /* Nonmatching */
+u8 daTag_Etc_c::getEventNo() {
+    return this->mBase.mParameters >> 0x18;
 }
 
 /* 00000084-00000090       .text getType2__11daTag_Etc_cFv */
-void daTag_Etc_c::getType2() {
-    /* Nonmatching */
+u32 daTag_Etc_c::getType2() {
+    return this->mBase.mParameters >> 8 & 0xF;
 }
 
+/* Not Matching */
 /* 00000090-000001B4       .text rangeCheck__11daTag_Etc_cFP10fopAc_ac_c */
-void daTag_Etc_c::rangeCheck(fopAc_ac_c*) {
-    /* Nonmatching */
+BOOL daTag_Etc_c::rangeCheck(fopAc_ac_c* pActor) {
+    BOOL result;
+    float fVar4;
+    float distanceMagnitude;
+    cXyz cxYz_planar_distance;
+    cXyz cXyz_distance;
+
+    cXyz_distance = ((pActor->current).pos - (this->current).pos);
```

Comment:
```suggestion
    delta = pActor->current.pos - current.pos;
```

typing `this->` isn't necessary, parentheses aren't necessary when order of operations is obvious, putting the type name in the variable name isn't necessary